Assets/Scripts/GameboardObjects/GameboardObjectManager.cs

```diff
@@ -380,6 +383,13 @@ public class GameboardObjectManager : NetworkSingleton<GameboardObjectManager>
             if (target != null && gbo.IsValidAttack(target) && !target.IsOwner && target.GetGboType() != Types.TRAP) {
                 if (!hitSpots.ContainsKey(raycastHitSpot)) hitSpots.Add(raycastHitSpot, new HitSpot(HexColors.VALID_ATTACK, gbo.GetOccupiedRadius()));
                 _isRaycastRangeValid = Gameboard.Instance.HighlightRaycastHitSpot(hitSpots);
+
+                points = new Transform[2];
+                    points[0] = _selectedGameboardObject.transform;
+                    points[1] = target.transform;
+                
+                lineScript.SetUpLine(points);
```

Comment:
Looks like this only shows up on the client that's attacking an enemy? This should instead be emitted on a ClientRPC so its shown on both sides